balances/src/lib.rs

```diff
@@ -243,6 +243,11 @@ impl<T: Config> Pallet<T> {
 	pub fn set_demurrage(cid: &CommunityIdentifier, demurrage: Demurrage) {
 		<DemurragePerBlock<T>>::insert(cid, &demurrage);
 	}
+
+	pub fn purge_balances(cid: CommunityIdentifier) -> DispatchResult {
+		<Balance<T>>::remove_prefix(cid, None);
+		Ok(())
+	}
```

Comment:
Why the return type if this cannot fail?

Reply:
haha just copy pasted from another function tbh :) makes sense!